### PR TITLE
Mattermost Apps: removed unnecessary steps from quick-start-go

### PR DESCRIPTION
**Pull request by levb**
#### Summary
Removed the steps that are no longer needed since the apps plugin is bundled with the server

#### Ticket Link
none

### CODE REVIEW

File: site/content/integrate/apps/quickstart/quick-start-serverless/_index.md [hunk 1]
```diff
@@ -30,32 +30,15 @@ See complete source code of the example [here](https://github.com/mattermost/mat
...
-Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`.
+Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides.
 
-In the System Console, ensure that the following are set to **true**:
+Once set up, open the System Console and ensure that the following are set to **true**:
 
 - `Enable Bot Account Creation`
 - `Enable OAuth 2.0 Service Provider`
 
 You also need at least `go1.16` installed. Please follow the guide [here](https://golang.org/doc/install) to install the latest version.
```
Comment (hanzei) [inline site/content/integrate/apps/quickstart/quick-start-serverless/_index.md:40]:
```suggestion
You also need at least `go1.18` installed. Please follow the guide [here](https://golang.org/doc/install) to install the latest version.
```

**Review by hanzei — APPROVED** [review]
Thanks for the cleanup

File: site/content/integrate/apps/quickstart/quick-start-go/_index.md [hunk 2]
```diff
@@ -16,32 +16,15 @@ This quick start guide explains the basics of writing a Mattermost app. In this
 
 ## Prerequisites
 
-Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`.
+Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides. 
 
-In the System Console, ensure that the following are set to **true**:
+Once set up, in the Mattermost System Console, ensure that the following are set to **true**:
 
 - `Enable Bot Account Creation`
```
Comment (cwarnermm) [inline site/content/integrate/apps/quickstart/quick-start-go/_index.md:23]:
```suggestion
- **Integrations > Enable Bot Account Creation**
```

File: site/content/integrate/apps/quickstart/quick-start-go/_index.md [hunk 3]
```diff
@@ -16,32 +16,15 @@ This quick start guide explains the basics of writing a Mattermost app. In this
...
 - `Enable OAuth 2.0 Service Provider`
```
Comment (cwarnermm) [inline site/content/integrate/apps/quickstart/quick-start-go/_index.md:24]: Is the admin able to configure this via the System Console?

File: site/content/integrate/apps/quickstart/quick-start-serverless/_index.md [hunk 4]
```diff
@@ -30,32 +30,15 @@ See complete source code of the example [here](https://github.com/mattermost/mat
 
 ## Prerequisites
 
-Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`.
+Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides.
 
-In the System Console, ensure that the following are set to **true**:
+Once set up, open the System Console and ensure that the following are set to **true**:
 
 - `Enable Bot Account Creation`
```
Comment (cwarnermm) [inline site/content/integrate/apps/quickstart/quick-start-serverless/_index.md:37]: Same feedback re: include path details to help admins find individual settings

**Review by cwarnermm — COMMENTED** [review]
Thanks, @levb! I've provided some feedback inline.

File: site/content/integrate/apps/quickstart/quick-start-serverless/_index.md (hunk 1 above)
Comment (neflyte) [reply site/content/integrate/apps/quickstart/quick-start-serverless/_index.md:40]: If we make this version change here, we should make it in all quick start guides.

Comment (cwarnermm) [reply site/content/integrate/apps/quickstart/quick-start-serverless/_index.md:40]: I agree, @neflyte! @levb - If you agree as well, would you be open to updating the remaining Go references for consistency and clarity?

Comment (hanzei) [reply site/content/integrate/apps/quickstart/quick-start-serverless/_index.md:40]: Maybe I'm missing something. Why is go needed for the other quickstart guides?

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

**Comment by levb** [comment]
Looks like @azigler's https://github.com/mattermost/mattermost-developer-documentation/pull/1106 actually took care of it, closing.

**Pull request closed without merging**